### ISSUE TITLE
Textarea field does not have the correct default height in the Wagtail page editor

### ISSUE THREAD

**Issue by onno-timmerman** [wagtail/wagtail#7519]
I notice that often the default height of the textarea is to small. If you translate a page and a textarea is multiline then the height look something like this. You need to enter the textarea and add a line to see it complete.

<img width="390" alt="" src="https://user-images.githubusercontent.com/178425/133407896-b5f8e869-ddff-4c02-a19e-c67419d6a68c.png">

**Comment by lb-** [comment]
We need to work out if this broke in 2.15 or if it is something that has always been an issue.

See https://github.com/wagtail/wagtail/pull/7375#issuecomment-902905005 

A refined way to initialise tab content was introduced in 2.15 that was meant to resolve the default height of text areas.

@onno-timmerman  - it is not super clear in your screenshot, is this a problem with StreamField text area or non StreamField? Also, are you aware of this having worked before and broke (it would be great to add some info about the versions you are using, including browser) so we can get to the bottom of this.

**Comment by onno-timmerman** [comment]
I can confirm this is the same for 2.15 it only is in streamfields. The textarea does not grow according the text in it.

**Comment by Scotchester** [comment]
I can confirm this bug as well, for both a standard TextBlock and a custom block where I've set the widget to AdminAutoHeightTextInput. I'm using Wagtail 2.15.1, and I see the bug in Firefox 91 (ESR), Chrome 98, and Safari 15.3 on macOS.

I can also confirm, for anyone who wants to hack around this until a fix is implemented in Wagtail, that @FlipperPA's [suggestion on this Stack Overflow answer](https://stackoverflow.com/a/69847831/279566) does the trick! I modified it to loop through all relevant `texareas`, resulting in:

```js
document.querySelectorAll('.widget-admin_auto_height_text_input textarea').forEach((textarea) => {
  const event = new Event('input');
  textarea.dispatchEvent(event);
})
```
